package.xml

```diff
@@ -2,7 +2,7 @@
   <name>rqt_topic</name>
   <version>1.1.0</version>
   <description>rqt_topic provides a GUI plugin for displaying debug information about ROS topics including publishers, subscribers, publishing rate, and ROS Messages.</description>
-  <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
+  <maintainer email="scott@openrobotics.org">Scott K Logan</maintainer>
```

Comment:
Usually we move the previous maintainers to the `<author>` list.

Reply:
528dc61